orcestra/postprocess/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "roll_threshold": 5,
+    "valid_radar_state": 13, 
+    "noise_threshold": 100,
+    "altitude_threshold": 4800
+}
```

Comment:
In general it is a nice idea to define thresholds that may be shared among several functions at a single point. However, maybe this should be an own `orcestra.constants` submodule. But I am also fine with merging this as is and refactor later on, if we agree that a different approach might have advantages

Reply:
Sounds good